zanata-war/src/main/java/org/zanata/action/HomePage.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2015, Red Hat, Inc. and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.zanata.action;
+
+import lombok.extern.slf4j.Slf4j;
+import org.jboss.seam.ScopeType;
+import org.jboss.seam.annotations.AutoCreate;
+import org.jboss.seam.annotations.In;
+import org.jboss.seam.annotations.Name;
+import org.jboss.seam.annotations.Observer;
+import org.jboss.seam.annotations.Scope;
+import org.zanata.ApplicationConfiguration;
+import org.zanata.events.HomeContentChangedEvent;
+
+/**
+ * @author Sean Flanigan <a href="mailto:sflaniga@redhat.com">sflaniga@redhat.com</a>
+ */
+@AutoCreate
+@Name("homePage")
+@Scope(ScopeType.APPLICATION)
+@Slf4j
+public class HomePage {
+
+    @In
+    private ApplicationConfiguration applicationConfiguration;
+
+    @In("commonMarkRenderer")
+    private CommonMarkRenderer renderer;
+
+    private String html;
+
+    public String getHtml() {
```

Comment:
Is `homePage.html` not enough?  That's how it will be used.

On 17 March 2015 at 11:54, Carlos Munoz <notifications@github.com> wrote:

> In zanata-war/src/main/java/org/zanata/action/HomePage.java
> <https://github.com/zanata/zanata-server/pull/732#discussion_r26544243>:
>
> > + */
> > +@AutoCreate
> > +@Name("homePage")
> > +@Scope(ScopeType.APPLICATION)
> > +@Slf4j
> > +public class HomePage {
> > +
> > +    @In
> > +    private ApplicationConfiguration applicationConfiguration;
> > +
> > +    @In("commonMarkRenderer")
> > +    private CommonMarkRenderer renderer;
> > +
> > +    private String html;
> > +
> > +    public String getHtml() {
>
> Can we call this something more descriptive? Perhaps mainContent or
> homeContent (since that's exactly what it's fetching)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/732/files#r26544243>.
>